src/AzureAuth/Ado/Constants.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Authentication.AzureAuth.Ado

```

Comment:
Note, this file is added in the prior PR, and will be merged indepednently first.